src/MDMMotionAnimator.m

```diff
@@ -16,24 +16,11 @@
 
 #import "MDMMotionAnimator.h"
 
-#import <UIKit/UIKit.h>
-
-#if TARGET_IPHONE_SIMULATOR
-UIKIT_EXTERN float UIAnimationDragCoefficient(void); // UIKit private drag coefficient.
-#endif
-
-static CGFloat simulatorAnimationDragCoefficient(void) {
-#if TARGET_IPHONE_SIMULATOR
-  return UIAnimationDragCoefficient();
-#else
-  return 1.0;
-#endif
-}
+#import "private/CABasicAnimation+MotionAnimator.h"
+#import "private/MDMUIKitValueCoercion.h"
+#import "private/MDMDragCoefficient.h"
 
-static CAMediaTimingFunction* timingFunctionWithControlPoints(CGFloat controlPoints[4]);
-static NSArray* coerceUIKitValuesToCoreAnimationValues(NSArray *values);
-static CABasicAnimation *animationFromTiming(MDMMotionTiming timing, CGFloat timeScaleFactor);
-static void makeAnimationAdditive(CABasicAnimation *animation);
+#import <UIKit/UIKit.h>
```

Comment:
Format/Nit: UIKit should be moved up to just after the related header.

Reply:
Done.